documentation/validation_log.rst

```diff
@@ -4,6 +4,20 @@ Validation Log
 Log
 ---
 
+
+Version: v
+~~~~~~~~~~~~~~~~
+
+Release Date: 
+'''''''''''''''''''''''''''''
+
+New features:
+'''''''''''''
+
+* morpho as git submodule
```

Comment:
Here there should be more informations about what this release is about: so much work done by all the SA group!!!!!!

Reply:
Thank you for your comments, Mathieu!  I just added more features to the validation_log.rst file and updated the mermithid version in Dockerfile and CmakeLists.txt.